packages/url_launcher/url_launcher/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,28 +1,41 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.flutter.plugins.urllauncherexample">
+  package="io.flutter.plugins.urllauncherexample">
 
-    <!-- The INTERNET permission is required for development. Specifically,
-         flutter needs it to communicate with the running application
-         to allow setting breakpoints, to provide hot reload, etc.
-    -->
-    <uses-permission android:name="android.permission.INTERNET"/>
+  <!-- io.flutter.app.FlutterApplication is an android.app.Application that
+       calls FlutterMain.startInitialization(this); in its onCreate method.
+       In most cases you can leave this as-is, but you if you want to provide
+       additional functionality it is fine to subclass or reimplement
+       FlutterApplication and put your custom class here. -->
+  <application
+    android:icon="@mipmap/ic_launcher"
+    android:label="url_launcher_example"
+    android:name="io.flutter.app.FlutterApplication">
+    <activity
+      android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+      android:hardwareAccelerated="true"
+      android:launchMode="singleTop"
+      android:exported="true"
+      android:name="io.flutter.plugins.urllauncherexample.EmbeddingV1Activity"
+      android:theme="@android:style/Theme.Black.NoTitleBar"
+      android:windowSoftInputMode="adjustResize">
+    </activity>
+    <activity
+      android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+      android:hardwareAccelerated="true"
+      android:launchMode="singleTop"
```

Comment:
@matthew-carroll seems to suggest this to be moved.
https://github.com/flutter/plugins/pull/2164#discussion_r334189140

Reply:
Done.